README.rst

```diff
@@ -302,24 +328,67 @@ proposal is ready for decision.
 
    * post their recommendation, with a rationale, on the GitHub discussion thread,
```

Comment:
Since evaluative discussion is now happening on the mailing list again, should this be on the mailing list instead?